package.json

```diff
@@ -134,7 +134,7 @@
     "@types/moment-duration-format": "^2.2.2",
     "@types/net-keepalive": "^0.4.0",
     "@types/nock": "^11.0.7",
-    "@types/node": "^10.17.20",
+    "@types/node": "^10.17.28",
```

Comment:
I think it's probably caused by the forceful breaking of npm-shrinkwrap rebase, normally an `npm i` can resolve package-lock/shrinkwrap gracefully (if it even needs to) but I think the forceful breakage makes that go all weird

Reply:
Yeah I've just checked and it's definitely due to breaking the rebase, anyway I don't actually need these changes as it turned out I just forget the requirements to make `util.promisify` type properly rather than it being out of date types